examples/autoencoder/README.md

```diff
@@ -90,6 +91,42 @@ After training for 30 epochs the loss function should look similar to this:
 
 ![Training loss](training-loss.png)
 
+## Model creation (Tensorflow)
+
+Select the dataset you just created and under the `Custom network` tab, select `Tensorflow`. There you can paste the following network definition:
```

Comment:
Please add a line that the below network was made using Tensorflow slim, but add that they could have done this using vanilla tensorflow, or even keras (as integrated into tensorflow)